### PR TITLE
[MINOR] feat(common): Support set args from SystemProperties

### PR DESCRIPTION
**Pull request by maobaolong**
### What changes were proposed in this pull request?

We can start server and specific arguments System.properties

### Does this PR introduce _any_ user-facing change?

No.

### How was this patch tested?

Add jvm args `-Drss.dashboard.http.port=19987` to start dashboard process, we can see it take effect through the log.

```
[2024-07-15 21:56:55.942] [main] [INFO] AbstractConnector.doStart - Started ServerConnector@26be6ca7{HTTP/1.1,[http/1.1]}{0.0.0.0:19987}
```

### CODE REVIEW

**Comment by rickyma** [comment]
Could you add some docs about this feature?

**Comment by zuston** [comment]
This looks the jvm properties is prior than the conf specified in the conf file

**Comment by maobaolong** [comment]
@zuston Yeah, this is what i want, I want to use config file as the template, but use jvm args to specific new value which can override the value set in the config file

**Comment by rickyma** [comment]
@maobaolong Please remove the auto-generated info in the description.

**Comment by maobaolong** [comment]
@rickyma Thanks for your remind, i updated the description, PTAL.

About the docs, would you mind I add it in the follow up PR?

**Comment by rickyma** [comment]
> About the docs, would you mind I add it in the follow up PR?

OK.

**Review by rickyma — APPROVED** [review]
LGTM.

File: common/src/main/java/org/apache/uniffle/common/config/RssBaseConf.java [hunk 1]
```diff
@@ -280,10 +281,14 @@ public class RssBaseConf extends RssConf {
...
     Map<String, String> properties = RssUtils.getPropertiesFromFile(fileName);
-
     if (properties == null) {
       return false;
     }
+    Map<String, String> systemProperties =
+        System.getProperties().stringPropertyNames().stream()
+            .collect(
+                Collectors.toMap(propName -> propName, propName -> System.getProperty(propName)));
+    properties.putAll(systemProperties);
```
Comment (xianjingfeng) [inline common/src/main/java/org/apache/uniffle/common/config/RssBaseConf.java:291]:
```suggestion
    System.getProperties().stringPropertyNames().stream().forEach(propName -> {
      properties.put( propName, System.getProperty(propName));
    });
```

**Review by zuston — APPROVED** [review]
LGTM. Thanks for your contribution @maobaolong . And thanks @rickyma @xianjingfeng for your review.

**Comment by maobaolong** [comment]
@xianjingfeng @zuston @rickyma Thanks for your review!

**Pull request merged**